src/roadmap/releases.md

```diff
@@ -193,15 +205,23 @@ Change the `generate_params.toml` file in **each** new chart as follows:
   1. If there's a `[workflowManager]` section, change `versionsApiURL` to
      `"https://versions.deis.com"` and `doctorApiURL` to `"https://doctor.deis.com"`
 
-Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` file:
+Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` and
+`workflow-$WORKFLOW_RELEASE/tpl/deis-controller-rc.yaml` files:
 
   1. Remove the `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` env var
 
+Change any "-dev" labels to reference the new release instead, and remove test wording:
+
+  1. Remove "for testing only" lines in any Chart.yaml or README.md
+  1. Replace "-dev" with the new release tag in those same files
+  1. Don't forget the e2e and router charts
+  1. Compare changes to the previous release charts to ensure consistency
+
 Commit and push your changes:
 
 ```bash
 git commit -a -m "chore(workflow-$WORKFLOW_RELEASE): releasing workflow-$WORKFLOW_RELEASE(-e2e)"
-git push origin HEAD:release-$WORKFLOW_RELEASE
+git push upstream HEAD:release-$WORKFLOW_RELEASE
```

Comment:
Yes, this assumes you have your `git` remotes set up in this convention. Having `origin` here tripped me up continually since the branch is expected to live at the deis/charts repository itself, so I think this is safer.